CMakeLists.txt

```diff
@@ -258,6 +270,10 @@ else( WIN32 ) # Apple AND Linux
 
 endif( WIN32 )
 
+if ( FULL_STATIC_BUILD )
+    set( CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -static -static-libstdc++ -static-libgcc" )
```

Comment:
Won't this break if FULL_STATIC_BUILD is set with MSVC for example?

Reply:
Will add condition, to make sure it won't get triggered for MSVC.